### ISSUE TITLE
php7 ?

### ISSUE THREAD

**Issue by flagoda** [wpkg/wpkg-express-ng#1]
Hi,
Does it work with PHP 7 ?  (debian 10)

**Comment by EvilFreelancer** [comment]
Hello! I hope so, but never tried :)

This project is optimized for PHP 5, so may be some issues.

As you probably noticed, the WPKG project is abandoned and has not been updated for a very long time, this is clearly visible on the download page https://wpkg.org/Download

wpkg-express-ng is also abandoned https://forge.probesys.com/projects/wpkg-express-ng/files

**Comment by flagoda** [comment]
Hello,
Yes i see this. This is just to migrate to Debian 10 (php7), the time to move to other solution.

**Comment by EvilFreelancer** [comment]
You may try to make Docker image with this system for example.

**Comment by flagoda** [comment]
Not work with PHP 7.4.3  :(  Stay on blank page
I think problem with .htaccess but not know enough php & cake

**Comment by EvilFreelancer** [comment]
I don;t think so, migration from PHP 5.6 to 7.0 is a big headache. For example refactoring of some database logic and many many more.
